kube/client.go

```diff
@@ -14,19 +14,35 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"log"
 	"net/http"
+	"net/url"
 	"os"
 	"path/filepath"
 	"sync"
 	"time"
+
+	"tailscale.com/util/multierr"
 )
 
 const (
 	saPath     = "/var/run/secrets/kubernetes.io/serviceaccount"
 	defaultURL = "https://kubernetes.default.svc"
 )
 
+// rootPathForTests is set by tests to override the root path to the
+// service account directory.
+var rootPathForTests string
```

Comment:
Package global means we can never parallelize any tests that use kube. It would be nice to avoid that, but given the current semantics of this package, I guess this is fine.

Reply:
yeah, i plan on fixing this ~soon.